Add support for loading files in BZip2 and XZ formats (#62)
JuliaData/RData.jl: README.md

For running R code from Julia see [RCall.jl](https://github.com/JuliaInterop/RCall.jl).

## Installation

From Julia REPL:
```julia
Pkg.add("RData")
```

### Compression formats

> Gzip compressed R data files are supported by default. Bzip2 and Xz compressed files are also available via optional dependencies on `CodecBzip2` or `CodecXz`.

Reviewer: ```suggestion
R data files could be compressed by either *Gzip* (the default), *Bzip2* or *Xz* methods. `RData.jl` supports *Gzip*-compressed files out-of-the-box. To read *Bzip2* or *Xz*-compressed files [CodecBzip2](https://github.com/bicycle1885/CodecBzip2.jl) or [CodecXz](https://github.com/bicycle1885/CodecXz.jl) must be installed.
```

> For example, if you need to load a file compressed with Bzip2 you must first install the required codec:

Reviewer: ```suggestion
For example, to load a file compressed by *Bzip2* you must first install the required codec:
```

```julia
Pkg.add("CodecBzip2")
```

> Then in your code ensure it is loaded before `RData` so the initializers in `RData` can detect it:

Reviewer: ```suggestion
Then ensure *CodecBzip2* is loaded before *RData*, so the initializers in *RData* can detect it:
```
